wolfcrypt/src/hmac.c

```diff
@@ -815,13 +815,13 @@ int wc_HKDF(int type, const byte* inKey, word32 inKeySz,
         return BAD_FUNC_ARG;
 
 #ifdef WOLFSSL_SMALL_STACK
-    tmp = (byte*)XMALLOC(MAX_DIGEST_SIZE, myHmac.heap, DYNAMIC_TYPE_TMP_BUFFER);
+    tmp = (byte*)XMALLOC(MAX_DIGEST_SIZE, NULL, DYNAMIC_TYPE_TMP_BUFFER);
```

Comment:
It looks like this section of code is with small stack and small stack is not currently supported with static memory. I think NULL is the right choice for the temporary buffer with small stack enabled.

myHmac.heap could be left in but if left would need initialized which would require changing the function prototype to accept a heap hint. It is best to follow the pattern of not supporting heap hints with pre existing functions when wrapped in the small stack macro.